weatherApp/App/Data/WeatherRepository.swift

```diff
@@ -70,6 +70,30 @@ class WeatherRepository {
         }
     }
     
+    func fetchCurrentWeather(coord: Coordinates, completion: @escaping ((CurrentWeatherCoreData?) -> Void)) {
+        switch reachability.connection {
+        case .unavailable:
+            completion(nil)
+            let currentWeatherCoreData = coreDataService.fetchCurrentWeather(coord: coord)
+            completion(currentWeatherCoreData)
+        default:
+            weatherService.fetchCurrentWeather(coord: coord) { [weak self] (currentWeather) in
+                guard
+                    let self = self,
+                    let currentWeather = currentWeather
+                else {
+                    completion(nil)
+                    return
+                }
+                let _ = self.coreDataService.createCurrentWeatherFrom(currentWeather: currentWeather)
```

Comment:
You can use @discardableResult on createCurrentWeatherFrom. With this, you don't need to use 
```
let _ =
```